content/blog/2023/02/2023-02-01-gsoc-update.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "GSoC 2023: \"On your marks, get set, ...\""
+tags:
+- gsoc
+- gsoc2023
+- community
+- events
+- developer
+authors:
+- jmMeessen
+description: >
+  The preparation phase for GSoC 2023 is nearly completed. This is what comes next.
+opengraph:
+  image: /images/gsoc/opengraph.png
+links:
+  discourse: https://community.jenkins.io/t/gsoc-2023-on-your-marks-get-set/5540
+---
+
+// image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=left]
+
+In a couple of days, the Jenkins GSoC 2023 preparation will culminate with our official Organization Application.
+It is now time to shed some light on what is coming next. 
+
+image:/images/gsoc/opengraph.png[Jenkins GSoC, role=center, float=center]
+
+Since December 2022, we actively prepared for the link:https://summerofcode.withgoogle.com/[2023 Google Summer of Code] (GSoC) edition for the Jenkins project. 
+This entailed collecting and discussing project ideas, setting up a mentoring team, and giving advise on how to build the necessary "GSoC Jenkins muscle".
+
+Our application will be submitted before February 7.
+On February 22, we will know if we are part of the adventure again.
+
+Although we are waiting for Google's decision, we will start the next GSoC phase without delay.
+In this phase, the candidates will prepare their detailed GSoC application with the help of the Jenkins Community.
+The proposals are based on the link:/projects/gsoc/2023/project-ideas/[2023 project ideas].
+The phase will culminate with the selection of the students/contributors participating in in this year's GSoC. 
+The deadline for the application is April 4.
+
+Read carefully the guidelines for a successful application on the link:/projects/gsoc/students/[Information for GSoC Contributors] page.
+
+In a nutshell, select a project idea and begin thinking on how you will build a proposal to convince the mentors that you are the best candidate to bring it to a successful end.
+
+We will help you through the currently available channels (Community forum and Gitter) but also with weekly on-line link:https://en.wiktionary.org/wiki/office_hours[office-hours].
+During these meetings you can ask for any clarification, disambiguation, precision on the project ideas or about the GSoC program.
+These sessions will be recorded so that everyone can benefit from them.
+
+Before setting up the weekly Office Hour, we will organize a poll to choose a timing that suits the timezones of most of the candidates.
+Last year we held sessions at 14h00 UTC for Europe and Asia and 03h00 UTC for the US and the Pacific.
+
+When the Jenkins participation to GSoC 2023 is confirmed you can actively start working on your draft proposal.
+Make these drafts available to the Jenkins Community for review via https://community.jenkins.io.
+The previous year's experience shows that proposals that didn't use this public discussion and review process turn out to be of to poor quality to be selected.
```

Comment:
```suggestion
When the Jenkins participation in GSoC 2023 is confirmed, you can actively start working on your draft proposal.
Make these drafts available to the Jenkins Community for review via link:https://community.jenkins.io[the community forum].
Previous years' experience shows that proposals not submitted for the public discussion and review process, are not selected due to being less convincing.
```
small suggestion for wording, just trying to word this idea so that it is clear the discussion and review are critical to success